tests/Tests/400__ods_dashboard/415__ods_dashboard_settings_pod_tolerations.robot

```diff
@@ -6,13 +6,25 @@ Suite Setup      Begin Web Test
 Suite Teardown   Teardown
 
 
+*** Variables ***
+${TOLERATION_CHECKBOX}=    //input[@id="tolerations-enabled-checkbox"]
+
+
 *** Test Cases ***
-Test Setting Pod Toleration Via UI
+Test Setting Unsupported Pod Toleration Via UI
     [Documentation]    Sets a Pod toleration via the admin UI
     [Tags]  Sanity    Tier1
-    ...     ODS-1684
+    ...     ODS-1788
     Menu.Navigate To Page    Settings    Cluster settings
     Wait Until Page Contains    Notebook pod tolerations
+    Set Pod Toleration Via UI    --UNSUPPORTED--
```

Comment:
If you want to enhance it a little bit more, consider creating a keyword `Unsupported Pod Toleration Should Not Be Allowed` (or similar) wrapping lines 20-22 and then calling it with different unsupported values

Reply:
Done